frontend/src/pages/premium.module.scss

```diff
@@ -212,7 +217,7 @@
 .bundle-banner-section {
   width: 100%;
   margin: 0 auto;
-  padding: $spacing-lg $spacing-md;
+  padding: calc($spacing-xl * 2) $spacing-md;
```

Comment:
nit/suggestion: Add a comment for this semi-magic number

Reply:
added a comment - using it in other locations too now after talking to design.